src/SWP/Bundle/MenuBundle/MenuEvents.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Superdesk Web Publisher Menu Bundle.
+ *
+ * Copyright 2017 Sourcefabric z.ú. and contributors.
+ *
+ * For the full copyright and license information, please see the
+ * AUTHORS and LICENSE files distributed with this source code.
+ *
+ * @copyright 2017 Sourcefabric z.ú
+ * @license http://www.superdesk.org/license
+ */
+
+namespace SWP\Bundle\MenuBundle;
+
+final class MenuEvents
+{
+    /**
+     * The MENU_CREATED event occurs after menu is created.
+     *
+     * @Event("Symfony\Component\EventDispatcher\GenericEvent")
+     *
+     * @var string
+     */
+    const MENU_CREATED = 'swp.menu.created';
+
+    /**
+     * The MENU_DELETED event occurs after menu is deleted.
+     *
+     * @Event("Symfony\Component\EventDispatcher\GenericEvent")
+     *
+     * @var string
+     */
+    const MENU_DELETED = 'swp.menu.deleted';
+
+    private function __construct()
```

Comment:
so the class won't be instantiated as its not needed to do it, it just keeps events as consts. Note the construct is private.